src/main/java/dev/task/dndquest/model/entity/Player.java

```diff
@@ -0,0 +1,57 @@
+package dev.task.dndquest.model.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.List;
+
+@Entity
+@Data
+@RequiredArgsConstructor
```

Comment:
not needed here

Reply:
I`m using constructor in dto mapper that uses only fields login and password